src/PerfView/GcStats.cs

```diff
@@ -543,14 +543,14 @@ public static void ToXmlAttribs(TextWriter writer, TraceProcess stats, TraceLoad
                     writer.WriteLine(">");
 
                     var sb = new System.Text.StringBuilder();
-                    for (var gens = Gens.Gen0; gens <= Gens.GenLargeObj; gens++)
+                    for (int gens = 0; gens < perHeapHistory.GenData.Length; gens++)
                     {
                         sb.Clear();
                         sb.Append("        ");
-                        writer.Write(perHeapHistory.GenData[(int)gens].ToXml(gens, sb).AppendLine().ToString());
+                        writer.Write(perHeapHistory.GenData[(int)gens].ToXml((Gens)gens, sb).AppendLine().ToString());
                     }
 
-                    writer.Write("      </PerHeapHistory>");
+                    writer.WriteLine("      </PerHeapHistory>");
```

Comment:
any reason to change this? it was more compact before

Reply:
I think it looks better that way, but I won't mind changing it back.